src/app/containers/ConsentBanner/index.jsx

```diff
@@ -1,11 +1,21 @@
 import React, { useContext } from 'react';
+import { oneOfType, func, shape, any } from 'prop-types';
 import { RequestContext } from '#contexts/RequestContext';
 import Canonical from './index.canonical';
 import Amp from './index.amp';
 
-const ConsentBanner = () => {
+const ConsentBanner = ({ onDismissFocusRef }) => {
   const { isAmp } = useContext(RequestContext);
-  return isAmp ? <Amp /> : <Canonical />;
+  return isAmp ? <Amp /> : <Canonical onDismissFocusRef={onDismissFocusRef} />;
```

Comment:
Is the focus management not needed for amp?

Reply:
On amp, the banner doesn't load with clientside js (as it is not supported on amp) so the bug doesn't occur on amp. The serverside rendered response includes the banner so NVDA successfully steps through the banner first without the need for focus managment.

The context here explains the bug a bit better: https://github.com/bbc/simorgh/issues/8855